doc/active_docs/Analytics API.json

```diff
@@ -266,12 +266,276 @@
+              "name": "format",
+              "dataType": "string",
+              "defaultValue": "json",
+              "required": true,
+              "paramType": "path",
+              "allowableValues": {
+                "values": [
+                  "json",
+                  "xml"
+                ],
+                "valueType": "LIST"
+              }
+            },
+            {
+              "name": "access_token",
+              "description": "A personal Access Token",
+              "dataType": "string",
+              "required": true,
+              "paramType": "query",
+              "threescale_name": "access_token"
+            },
+            {
+              "name": "backend_api_id",
+              "description": "ID of the Backend.",
+              "dataType": "int",
+              "required": true,
+              "paramType": "path"
+            },
+            {
+              "description": "System name of metric to get data for.",
+              "name": "metric_name",
+              "dataType": "string",
+              "required": true,
+              "paramType": "query",
+              "threescale_name": "backend_api_metric_names"
+            },
+            {
+              "name": "since",
+              "description": "Time range start. Format YYYY-MM-DD HH:MM:SS, '2012-02-22', '2012-02-22 23:49:00'.",
+              "dataType": "string",
+              "required": true,
+              "paramType": "query"
+            },
+            {
+              "name": "period",
+              "description": "Period combined with since time gives stats for the time range [since .. since + period]. It is required if until time is not passed.",
+              "dataType": "string",
+              "required": false,
+              "paramType": "query",
+              "defaultValue": "",
+              "allowableValues": {
+                "values": [
+                  "year",
+                  "month",
+                  "week",
+                  "day"
+                ],
+                "valueType": "LIST"
+              }
+            },
+            {
+              "name": "until",
+              "description": "Time range end. Format YYYY-MM-DD HH:MM:SS",
+              "dataType": "string",
+              "required": false,
+              "paramType": "query"
+            },
+            {
+              "name": "granularity",
+              "description": "Granularity of results, each period has an associated granularity.",
+              "dataType": "string",
+              "required": true,
+              "paramType": "query",
+              "defaultValue": "month",
+              "allowableValues": {
+                "values": [
+                  "month",
+                  "day",
+                  "hour"
+                ],
+                "valueType": "LIST"
+              }
+            },
+            {
+              "description": "Time zone for calculations.",
```

Comment:
I do not think that this belongs to this PR so we should probably run `bundle exec rake doc:swagger:generate:all` before this PR 😂

Reply:
My bad! It DOES belong to this PR! 👍